packages/backend/src/open_payments/payment/outgoing/routes.ts

```diff
@@ -100,9 +102,18 @@ async function createOutgoingPayment(
   deps: ServiceDependencies,
   ctx: CreateContext<CreateBody>
 ): Promise<void> {
+  const walletAddress = await deps.walletAddressService.getOrPollByUrl(
```

Comment:
Do we want to potentially poll here? Typically, the polling probably necessary only on the receiving side, i.e. incoming payment creation only. (since that is the most likely place where we would need to allow the ASE to make a request to their backend to create the corresponding wallet addresses.